pom.xml

```diff
@@ -135,69 +140,67 @@
 			</exclusions>
 		</dependency>
 
+		<!-- used to get Google Maps' services to get distance or localization
+			  -->
+		<dependency>
+			<groupId>com.google.maps</groupId>
+			<artifactId>google-maps-services</artifactId>
+			<version>0.11.0</version>
+		</dependency>
+
+		<!-- Dependencies used for Unit Tests -->
 		<dependency>
 			<groupId>org.junit.platform</groupId>
 			<artifactId>junit-platform-launcher</artifactId>
 			<version>1.4.2</version>
 			<scope>test</scope>
 		</dependency>
-
 		<dependency>
-			<groupId>com.google.maps</groupId>
-			<artifactId>google-maps-services</artifactId>
-			<version>0.9.0</version>
-		</dependency>
-
-		<dependency>
-			<groupId>com.google.code.gson</groupId>
-			<artifactId>gson</artifactId>
-			<version>2.8.5</version>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-api</artifactId>
+			<version>5.4.2</version>
+			<scope>test</scope>
 		</dependency>
-
 		<dependency>
-			<groupId>com.squareup.okhttp3</groupId>
-			<artifactId>okhttp</artifactId>
-			<version>3.14.2</version>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>5.4.2</version>
+			<scope>test</scope>
 		</dependency>
 
+		<!--  Dependencies used by JSON Convert to manipulate JSONB -->
 		<dependency>
-			<groupId>joda-time</groupId>
-			<artifactId>joda-time</artifactId>
-			<version>2.10.2</version>
+			<groupId>javax.json.bind</groupId>
+			<artifactId>javax.json.bind-api</artifactId>
+			<version>1.0</version>
 		</dependency>
-
+		<!--  https://projects.eclipse.org/projects/ee4j.yasson -->
 		<dependency>
-			<groupId>org.joda</groupId>
-			<artifactId>joda-convert</artifactId>
-			<version>2.2.1</version>
+			<groupId>org.eclipse</groupId>
+			<artifactId>yasson</artifactId>
+			<version>1.0</version>
 		</dependency>
-		
-		<!-- à enlever aprés -->
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
-			<version>4.12</version>
-			<scope>test</scope>
-			<type>jar</type>
+			<groupId>org.glassfish</groupId>
+			<artifactId>javax.json</artifactId>
+			<version>1.1.4</version>
+			<scope>runtime</scope>
 		</dependency>
-		
 		<dependency>
-    			<groupId>org.junit.jupiter</groupId>
-    			<artifactId>junit-jupiter-engine</artifactId>
-    			<version>5.4.2</version>
-    			<scope>test</scope>
+			<groupId>javax.json</groupId>
+			<artifactId>javax.json-api</artifactId>
+			<version>1.1</version>
 		</dependency>
-
 	</dependencies>
-	
+
 	<build>
 		<plugins>
 			<plugin>
-			    <artifactId>maven-surefire-plugin</artifactId>
-			    <version>2.22.1</version>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>3.0.0-M4</version>
```

Comment:
Pas de bonne raison d’utiliser cette version. Utilisez plutôt la dernière qui ne se termine pas pas M* (indique des versions non encore finalisées).

Reply:
changes done in commit a78ac897c70df8010542c3942d293874e08948ee

Reply:
new Version is 2.22.2